boxsdk/object/base_object.py

```diff
@@ -117,7 +117,7 @@ def update_info(self, data, params=None, headers=None, **kwargs):
             Update the object attributes in data.keys(). The semantics of the
             values depends on the the type and attributes of the object being
             updated. For details on particular semantics, refer to the Box
-            developer API documentation <developers.box.com/docs>.
+            developer API documentation <https://box-content.readme.io/docs/>.
```

Comment:
Is this what we want? That leads to a getting started page. Should we link to https://box-content.readme.io/reference, or even just to https://box-content.readme.io (which redirects to https://box-content.readme.io/reference) instead?